packages/docusaurus-plugin-google-analytics/src/index.js

```diff
@@ -53,6 +53,7 @@ module.exports = function(context) {
             innerHTML: `
               window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
               ga('create', '${trackingID}', 'auto');
+              ga('set', 'anonymizeIp', ${anonymizeIP});
```

Comment:
It probably makes sense to do an input processing for Google Analytics - do not to set `anonymizeIP` if it is not define in config.

Reply:
I was being a bit lazy because the string templating here is not very nice to deal with. I guess I should do that just in case.